### PR TITLE
Feature/add translation fr

### PR DESCRIPTION
**Pull request by canssens**
Hello Antash,

I added the middleware to translate the website.
I changed all templates to translate the english.

I generated french and russian, and i compilted these languages. 
French should be almost done, and i did in russian only the sentences included in the exercisync version.

So if the merge and test are OK, the version for Russian people should be exactly the same, for english it will be full english, and for French in french.

I joined the two commands useful for translation in the readme.
To develop, I used this guide : https://medium.com/@nolanphillips/a-short-intro-to-translating-your-site-with-django-1-8-343ea839c89b
Last thing, it requires a new package on the server : gettext.

### CODE REVIEW

File: tapiriik/services/GarminConnect/garminconnect.py [hunk 1]
```diff
@@ -133,6 +134,7 @@ def __init__(self):
         # Ensure the rate lock file exists (...the easy way)
         open(rate_lock_path, "a").close()
         self._rate_lock = open(rate_lock_path, "r+")
+        '''
```
Comment (Antash) [inline tapiriik/services/GarminConnect/garminconnect.py]: Why do we need to comment this block?

**Review by Antash — APPROVED** [review]
Thank you Charles, great job!
One more question: do we really need to put .mo binary files under source control?

File: tapiriik/services/GarminConnect/garminconnect.py (hunk 1 above)
Comment (canssens) [reply tapiriik/services/GarminConnect/garminconnect.py]: My bad, i had to comment it to launch locally tapiriik, i forgot to uncomment

**Comment by canssens** [comment]
We could remove the compiled file mo. We have just to launch the compile at initialization.

**Pull request merged**